src/coreclr/jit/emitarm64.cpp

```diff
@@ -9746,6 +9760,31 @@ void emitter::emitIns_R_R_I(instruction     ins,
             }
             break;
 
+        case INS_sve_mov:
+        case INS_sve_cpy:
+            optionalShift = true;
+            assert(insOptsScalableStandard(opt));
+            assert(isVectorRegister(reg1));    // DDDDD
+            assert(isPredicateRegister(reg2)); // GGGG
+            if (!isValidSimm8(imm))
+            {
+                assert(isValidSimm8_MultipleOf256(imm));
+                assert(insOptsScalableAtLeastHalf(opt));
+                hasShift = true;
+                imm      = imm / 256;
+            }
+            if (sopt == INS_SCALABLE_OPTS_PREDICATE_MERGE)
+            {
+                fmt = IF_SVE_BV_2A_J;
+            }
+            else
+            {
+                fmt = IF_SVE_BV_2A;
```

Comment:
assert of what the value of `sopt` should be here?

Reply:
Done